holidays/countries/finland.py

```diff
@@ -97,6 +102,83 @@ def _populate_public_holidays(self):
         # Second Day of Christmas.
         self._add_christmas_day_two(tr("Tapaninpäivä"))
 
+    def _populate_unofficial_holidays(self):
+        if self._year >= 1854:
+            # Runeberg Day
+            self._add_holiday_feb_5(tr("Runebergin päivä"))
+
+        if self._year >= 1860:
+            # Kalevala Day, Day of Finnish Culture
+            self._add_holiday_feb_28(tr("Kalevalan päivä, suomalaisen kulttuurin päivä"))
+
+        if self._year >= 2007:
+            #  Minna Canth Day, Day of Equality
+            self._add_holiday_mar_19(tr("Minna Canthin päivä, tasa-arvon päivä"))
+
+        if self._year >= 1978:
+            # Mikael Agricola Day, Day of the Finnish Language
+            self._add_holiday_apr_9(tr("Mikael Agricolan päivä, suomen kielen päivä"))
+
+        if self._year >= 1987:
+            # National War Veterans' Day
+            self._add_holiday_apr_27(tr("Kansallinen veteraanipäivä"))
+
+        if self._year >= 2019:
+            # Europe Day
+            self._add_holiday_may_9(tr("Eurooppa-päivä"))
+
+        if self._year >= 1918:
+            # Mothers' Day
+            self._add_holiday_may_12(tr("Äitienpäivä"))
```

Comment:
```suggestion
            self._add_holiday_2nd_sun_of_may(tr("Äitienpäivä"))
```
It's Second Sunday in May, not a fixed date.